README.md

```diff
@@ -70,40 +71,48 @@ source ./venv/bin/active;
 
 Note: There is a flag which ends up being observed for any of the following commands which start the postgres database:
 
-`-r / --ram-disk` will cause the database to back its "storage" on a RAM disk instead of non-volatile storage. This has the upside of being much faster as there is essentially no limit to the IOPS available to the database, but the data won't be able to survive a restart and will require being `replicate-db`'d back into place. 
-
-The default is to use the disk in the host to back the database, which is the operation our team is most familiar with, so if you don't need or want the RAM disk configuration, you can ignore this option. 
+`-r / --ram-disk` will cause the database to back its "storage" on a RAM disk instead of non-volatile storage. This has the upside of being much faster as there is essentially no limit to the IOPS available to the database, but the data won't be able to survive a restart and will require being `replicate-db`'d back into place.
 
+The default is to use the disk in the host to back the database, which is the operation our team is most familiar with, so if you don't need or want the RAM disk configuration, you can ignore this option.
 
 #### `vision-zero build`
-Rebuild the stack's images based on the Dockerfiles found in the repository. They are built with the `--no-cache` flag which will make the build process slower, but avoid any stale image layers that have inadvertently cached out-of-date apt resource lists. 
+
+Rebuild the stack's images based on the Dockerfiles found in the repository. They are built with the `--no-cache` flag which will make the build process slower, but avoid any stale image layers that have inadvertently cached out-of-date apt resource lists.
 
 #### `vision-zero db-up` & `vision-zero db-down`
+
 Start and stop the postgres database
 
 #### `vision-zero graphql-engine-up` & `vision-zero graphql-engine-down`
+
 Start and stop the Hasura graphql-engine software
 
 #### `vision-zero vze-up` & `vision-zero vze-down`
+
 Start and stop the Vision Zero Editor
 
 #### `vision-zero vzv-up` & `vision-zero vzv-down`
+
 Start and stop the Vision Zero Viewer
 
 #### `vision-zero psql`
+
 Start a `psql` postgreSQL client connected to your local database
 
 #### `vision-zero tools-shell`
+
 Start a `bash` shell on a machine with supporting tooling
 
 #### `vision-zero stop`
+
 Stop the stack
 
 #### `vision-zero replicate-db`
-* Download a snapshot of the production database
-* Store the file in `./atd-vzd/snapshots/visionzero-{date}-{with|without}-change-log.sql
-* Drop local `atd_vz_data` database
-* Create and repopulate the database from the snapshot
+
+- Download a snapshot of the production database
+- Store the file in `./database/snapshots/visionzero-{date}-{with|without}-change-log.sql
+- Drop local `atd_vz_data` database
+- Create and repopulate the database from the snapshot
```

Comment:
Linting. I'm leaving it because it's not wrong, but I personally prefer the completely syntactically allowed `*` as my UL LI marker.